content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -132,6 +132,29 @@ directory should be rotated. In Kubernetes clusters brought up by
 the `kube-up.sh` script, those logs are configured to be rotated by
 the `logrotate` tool daily or once the size exceeds 100MB.
 
+## Node Log Viewer
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+Kubernetes v1.24 offers initial support for viewing node logs using kubectl;
+this is an alpha feature that is disabled by default. You can enable the alpha
```

Comment:
Isn't the fact that an alpha feature is turned off by default commonplace knowledge now?

```suggestion
This is an alpha feature and can be enabled
```